app/controllers/api/v1/security.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'doorkeeper/grape/helpers'
+
+module API
+  module V1
+    class Security < Grape::API
+      helpers Doorkeeper::Grape::Helpers
+
+      before do
+        doorkeeper_authorize! unless ENV['ALLOW_SIMPLE_LOGIN'] == 'true'
+        def current_account
+          if ENV['ALLOW_SIMPLE_LOGIN'] == 'true'
```

Comment:
do you have more details about ALLOW_SIMPLE_LOGIN I am not in favor of merging this
Also I would only merge the patch provided some unit tests you need to find a work around.